pkg/cmd/cli/secrets/add_secret_to_obj.go

```diff
@@ -61,14 +59,14 @@ type AddSecretOptions struct {
 // NewCmdAddSecret creates a command object for adding a secret reference to a service account
 func NewCmdAddSecret(name, fullName string, f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	o := &AddSecretOptions{Out: out}
-	typeFlag := "mount"
+	typeFlags := util.StringList{"mount"}
```

Comment:
pretty sure this default value will always be in the list... you need to make the default blank and treat an empty list like "mount"